vowpalwabbit/cb_adf.cc

```diff
@@ -261,6 +263,11 @@ void cb_adf::learn_MTR(multi_learner& base, multi_ex& examples)
   examples[_gen_cs.mtr_example]->num_features = nf;
   examples[_gen_cs.mtr_example]->weight = old_weight;
   std::swap(_gen_cs.mtr_ec_seq[0]->pred.a_s, _a_s_mtr_cs);
+  if (_ccb)
+  {
+    examples[0]->l.conditional_contextual_bandit.explicit_included_actions.clear();
```

Comment:
Could re-use `reduction_features` to communicate extra information between reductions that don't fit in the label or prediction and explicitly extend `ccb_reduction_features.h` to hold a field specifically for this

Reply:
I would say that modifying the reduction features of a problem type other than yourself is also a break of label/prediction layering rules

Reply:
hm so then the only way is via the `pred.a_s` but somehow making it clear that it's not a prediction?

Reply:
Or by returned a different prediction type when this mode is enabled

Reply:
This step isn't necessary and actually messed with some other CCB functionality. The cb labels and predictions are already in ccb so we dont need to do anything here